server/src/main/java/org/opentosca/toscana/api/CsarController.java

```diff
@@ -160,14 +153,9 @@ This Request (Supporting Post and Put methods) uploads a csar to the transformer
     @ApiResponses({
         @ApiResponse(
             code = 201,
-            message = "The upload and parsing of the csar was sucessful",
+            message = "The upload of the csar was successful",
             response = Void.class
         ),
-        @ApiResponse(
-            code = 400,
```

Comment:
is this Method only lauching a upload job now? If not: what is returned if the upload fails?

Reply:
Csar upload now will return 201 even if csar validation fails. In the old implementation, code 400 would only get returned if the csar validation failed. I eliminated this possibility, so I also removed return code 400.

Reply:
Okay.